chapter-template/release-notes-template.md

```diff
@@ -28,7 +28,8 @@ SystemLink Enterprise includes a deployment of the [RabbitMQ](https://www.rabbit
 | RabbitMQ Version | First SystemLink Enterprise Version | Last SystemLink Enterprise Version |
 |------------------|-------------------------------------|------------------------------------|
 | 3.11.x           | 0.12.x                              | 0.15.x                             |
-| 3.12.x           | 0.16.x                              | current                            |
+| 3.12.x           | 0.16.x                              | 0.24.x                             |
+| 3.12.x           | 0.25.x                              | current                            |
```

Comment:
Shouldn't this RabbitMQ version be 3.13.x here?

Reply:
Ugh, yes